### PR TITLE
Dmceachernmsft/meetingCompositeLocalesIcons

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What
Add localization Strings for meeting composite specific buttons. Break up meeting composite to make more readable when wrapped by a basic composite.

# Why
works toward larger localization support for the meeting composite.

https://skype.visualstudio.com/SPOOL/_workitems/edit/2648151

# How Tested
Ran locally in story book with different locals to see if strings changed appropriately. More work to come for specific meeting buttons and screens.

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/communication-react/review/communication-react.api.md [hunk 1]
```diff
@@ -1439,6 +1440,15 @@ export interface MeetingCompositeProps extends BaseCompositeProps<CallCompositeI
     meetingInvitationURL?: string;
 }
 
+// @public
+export interface MeetingCompositeStrings {
+    chatButtonLabel: string;
+    chatPaneTitle: string;
+    peopleButtonLabel: string;
+    peoplePaneSubHeader: string;
```
Comment (prprabhu-ms) [inline packages/communication-react/review/communication-react.api.md:1448]: nit: 
Title vs SubHeader. Should this be SubTitle?

**Review by prprabhu-ms — APPROVED** [review]

File: packages/communication-react/review/communication-react.api.md (hunk 1 above)
Comment (anjulgarg) [reply packages/communication-react/review/communication-react.api.md:1448]: +1 to SubTitle

File: samples/Chat/src/app/App.tsx [hunk 2]
```diff
@@ -16,7 +15,6 @@ console.info(`Thread chat sample using @azure/communication-chat : ${getChatSDKV
 console.info(`Build Date : ${getBuildTime()}`);
 
 initializeIcons();
-registerIcons({ icons: DEFAULT_COMPONENT_ICONS });
```
Comment (anjulgarg) [inline samples/Chat/src/app/App.tsx:19]: Why is it not needed anymore?

Comment (anjulgarg) [reply samples/Chat/src/app/App.tsx:19]: Is it because, composites registerIcons be default?

**Review by anjulgarg — APPROVED** [review]

File: samples/Chat/src/app/App.tsx (hunk 2 above)
Comment (dmceachernmsft) [reply samples/Chat/src/app/App.tsx:19]: Yeah the base composite registers them anyway and then takes in the icons its overriding from the prop. this causes the overriden icons to be hidden if left in.

**Pull request merged**